scripts/sanitycheck

```diff
@@ -1771,11 +1765,10 @@ class TestCase:
         achtung_regex = re.compile(
             br"(#ifdef|#endif)",
             re.MULTILINE)
-        warnings = None
 
         with open(inf_name) as inf:
-            with contextlib.closing(mmap.mmap(inf.fileno(), 0, mmap.MAP_PRIVATE,
```

Comment:
Wouldn't it be simpler to create a variable mmap_flags that you set conditionally for UNIX or Windows and then keep the code as-is without having to move it to a nested function and having to have 2 calls to mmap()?

Reply:
yes, I'll update.